es6/main.js

```diff
@@ -31,25 +33,29 @@ class WarpedTime {
         if (warped_time !== undefined) {
             this.setWarpedTime(warped_time)
         }
+        this.genesis_time = genesis_time || this.getWarpedTime()
+        this.most_future_time = this.getWarpedTime()
 
         if (store) {
             this.store.subscribe(this.handleStateChange.bind(this))
         }
     }
 
     setSpeed(speed) {
+        raise_if_not_number(speed, '@WarpedTime.setSpeed')
```

Comment:
You don't need to record the call stack manually in a string, you can use `(new Error()).stack` for that.